data/json/monsterdrops/zombie_cop.json

```diff
@@ -79,5 +79,121 @@
     "id": "cop_weapons",
     "type": "item_group",
     "items": [ { "group": "cop_melee", "prob": 60 }, { "group": "guns_cop", "prob": 40 } ]
+  },
+  {
+    "id": "mon_zombie_swat_death_drops",
+    "type": "item_group",
+    "subtype": "collection",
+    "magazine": 100,
+    "ammo": 40,
+    "entries": [
+      { "group": "swat_zombie_gear", "prob": 100, "damage": [ 0, 2 ] },
+      { "group": "swat_gloves", "prob": 30, "damage": [ 0, 4 ] },
+      { "group": "swat_helmet", "prob": 20, "damage": [ 0, 4 ] },
+      { "group": "swat_pants", "damage": [ 0, 4 ] },
+      { "group": "swat_shoes", "damage": [ 0, 4 ] },
+      { "group": "swat_torso", "damage": [ 0, 4 ] },
+      { "group": "swat_weapons", "prob": 50, "damage": [ 0, 1 ], "dirt": [ 0, 7050 ] },
+      { "group": "underwear", "damage": [ 0, 4 ] },
+      { "group": "clothing_glasses", "prob": 5 },
+      { "group": "clothing_watch", "prob": 10 },
+      { "item": "badge_swat", "prob": 10 },
+      { "item": "cash_card", "charges-min": 0, "charges-max": 50000, "prob": 50 }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "swat_zombie_gear",
+    "subtype": "collection",
+    "entries": [
+      {
+        "collection": [ { "item": "bandages", "prob": 60 }, { "item": "pocket_firstaid", "prob": 40 }, { "item": "1st_aid", "prob": 20 } ],
```

Comment:
A first aid manual is out of place here.

Reply:
Oh so that's what the "pocket_firstaid" is, whoops. I copied it from the cop loot table without double checking what it was... my bad, and you're right it probably shouldn't be there ^_^